src/main/kotlin/at/yawk/katbot/action/NCov.kt

```diff
@@ -66,8 +66,8 @@ class NCov @Inject constructor(private val eventBus: EventBus) {
                 var text = "${region.name} ${region.cases}"
 
                 val extra = ArrayList<String>()
-                if (region.deaths != 0) extra.add("${region.deaths} dead")
-                if (region.recoveries != 0) extra.add("${region.recoveries} recovered")
+                if (region.deaths != 0) extra.add("${region.deaths} ✝")
+                if (region.recoveries != 0) extra.add("${region.recoveries} 🎉")
```

Comment:
Unicode Icon should be this one (proposed by @poempelfox):
https://graphemica.com/🎉